src/icastats.c

```diff
@@ -291,7 +291,7 @@ int main(int argc, char *argv[])
 		return EXIT_SUCCESS;
 	} else if(delete){
 		stats_mmap(user);
-		stats_munmap(SHM_DESTROY);
+		stats_munmap(user, SHM_DESTROY);
 		return EXIT_SUCCESS;
 	}
 	if(all){
```

Comment:
I think, that this also fixes a bug. 

`stats_mmap(user)` will create a shm-segment with the filename `icastats_<user>` Without the fix, the following `stats_munmap(SHM_DESTROY)` will unlink the shm-segment with the filename `icastats_<euid>`. This is (IMHO) an error in cases, where `user != euid`. Or did I missed something?